### ISSUE TITLE
Controlling 3-4 big units.

### ISSUE THREAD

**Issue by konstantys** [Anuken/Mindustry-Suggestions#2332]
**Describe what you would like changed, and why.**
The option to control multiple units at the same time, with other units following you without logic, is a great one. Yet, sometimes it can be a little annoying. I am talking about the case when you control big units in the quantity of 3-4 (excluding your base unit). You will find yourself bumping into the other controlled units, making it hard to move. Consider controlling 3 other quads as a quad as an example. Travelling into some directions is made considerably slower. Thus, in my opinion this should get fixed. 

**Describe the changes you want to propose. Include possible alternatives.**
I have come up with two possible solutions; one of them, which is a more complicated one, would be improving following units' AI so they don't collide with each other and with the player. The other, easier to make option, would be increasing the range the units follow you at in the case of 3-4 units.

**Comment by RedstonerKBC** [comment]
I've been complaining about bumping units since v6 first came out. I'm not sure if it's just too hard of a thing to implement or if it's not that important or what. All I know is nothing has been done about it.

**Comment by Legomonster33** [comment]
this is more of a bug tbh

**Comment by stuffyAI** [comment]
> this is more of a bug tbh

Take a look at [the other AI suggestion](https://github.com/Anuken/Mindustry-Suggestions/issues/2334#issuecomment-823810705)

Tl;dr: Ai, like pathfinding, is expensive, depending on algorithm used, for memory (RAM) and processor (CPU). 
Since Mindustry is available for mobile devices, and Anuken somewhat announced thoughts to make it paid (see trello and/or discord), we really have to keep the limitations of mobile users in mind.

To be more specific, it is already hard enough to efficiently handle path-finding units of different sizes blocking each other (see starcraft dev talks), but this suggestion also requests handling should the user control one of the flock entities.
It is not a bug, but rather a limitation, hence such static formations are currently used.

I prefer the first solution, since the latter one may also yield undesired gameplay effects should units part too far from the commander
